src/lsp/cobol_lsp/lsp_request.ml

```diff
@@ -722,17 +751,21 @@ let on_request
         Ok (handle_folding_range registry params, state)
     | Shutdown ->
         Ok (handle_shutdown registry, ShuttingDown)
-    | DocumentSymbol  (* DocumentSymbolParams.t.t *) params ->
+    | DocumentSymbol params ->
         Ok (handle_document_symbol registry params, state)
     | TextDocumentCodeLens (* CodeLensParams.t.t *) params ->
         Ok (handle_codelens registry params, state)
+    | TextDocumentRename params ->
+        begin match handle_rename registry params with
+          | Ok workspaceEdit -> Ok (workspaceEdit, state)
+          | Error _ -> Error (CopybookRenamingForbidden)
+        end
```

Comment:
```suggestion
        Ok (handle_rename registry params, state)
```
and then use `Lsp_io.notify_error "Ignored renaming of a reference that occurs in a copybook"` in `handle_rename`.